DatadogTrace/Sources/Utils/ActiveSpansPool.swift

```diff
@@ -57,7 +57,6 @@ internal class ActiveSpansPool {
         rlock.unlock()
     }
 
-#if DD_SDK_COMPILED_FOR_TESTING
```

Comment:
I haven't find a way to define compilation flags for testing only in SPM. This method is internal anyway.